### PR TITLE
Changes the SSRM patrol ship's shuttle docking time to match every other shuttle.

### PR DESCRIPTION
**Pull request by Snowy1237**
See title.

### CODE REVIEW

**Comment by Snowy1237** [comment]
!review

**Comment by Snowy1237** [comment]
!bugfix

**Comment by FluffyGhoster** [comment]
This isn't a bug and will need to go through the usual approval process, the system is working as intended

File: html/changelogs/snowy1237-ssrm_shuttle_speed_fix.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - bugfix: "Fixes the SSRM patrol ship's shuttle docking time being way too slow."
```
Comment (FluffyGhoster) [inline html/changelogs/snowy1237-ssrm_shuttle_speed_fix.yml:58]:
```suggestion
  - qol: "Changed the SSRM patrol ship's shuttle docking time to speed it up."
```

**Review by FluffyGhoster — CHANGES_REQUESTED** [review]

**Comment by Snowy1237** [comment]
The docking speed for every other shuttle is 20 seconds, and I believed this was a bug. I will change the changelog, then.

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**